### PR TITLE
Add custom image ring to avatar

### PR DESCRIPTION
**Pull request by joannaquu**
### Platforms Impacted
- [x] iOS
- [x] macOS
- [x] win32 (Office)
- [ ] windows
- [ ] android

### Description of changes

Added option to add custom image ring to avatar

### Verification

Simulator Screen Shot - iPad Pro (12 9-inch) (5th generation) - 2022-08-01 at 11 33 42

This has been tested in iOS, macOS, and win32

### Pull request checklist

This PR has considered (when applicable):
- [ ] Automated Tests
- [x] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

**Comment by Saadnajmi** [comment]
@ValentinaKozlova FYI. We might do try and merge this in next few weeks as a new feature :)

**Comment by ValentinaKozlova** [comment]
> @ValentinaKozlova FYI. We might do try and merge this in next few weeks as a new feature :)

It looks nice :)

File: packages/components/Avatar/src/Avatar.styling.ts [hunk 1]
```diff
@@ -157,6 +157,20 @@ export const stylingSettings: UseStylingOptions<AvatarProps, AvatarSlotProps, Av
       },
       ['size', 'ringColor', 'ringBackgroundColor', 'ringThickness', ...borderStyles.keys],
     ),
+    imageRing: buildProps(
```
Comment (ValentinaKozlova) [inline packages/components/Avatar/src/Avatar.styling.ts:160]: it's interesting for me if it's possible to use the same ring slot and just to change tokens

Comment (ValentinaKozlova) [reply packages/components/Avatar/src/Avatar.styling.ts:160]: Oh, I see there's an image slot, so it's good to use a separate slot

**Pull request closed without merging**